from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Predecir(object):
    def setupUi(self, Form):
        Form.setObjectName("Form")
        Form.resize(570, 557)
        self.verticalLayout = QtWidgets.QVBoxLayout(Form)
        self.verticalLayout.setObjectName("verticalLayout")
        self.groupBox = QtWidgets.QGroupBox(Form)
        self.groupBox.setFlat(True)
        self.groupBox.setObjectName("groupBox")
        self.verticalLayout_2 = QtWidgets.QVBoxLayout(self.groupBox)
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.frame = QtWidgets.QFrame(self.groupBox)
        self.frame.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame.setFrameShadow(QtWidgets.QFrame.Plain)
        self.frame.setObjectName("frame")
        self.verticalLayout_3 = QtWidgets.QVBoxLayout(self.frame)
        self.verticalLayout_3.setObjectName("verticalLayout_3")
        self.label_2 = QtWidgets.QLabel(self.frame)
        self.label_2.setFrameShadow(QtWidgets.QFrame.Plain)
        self.label_2.setAlignment(QtCore.Qt.AlignCenter)
        self.label_2.setObjectName("label_2")
        self.verticalLayout_3.addWidget(self.label_2)
        self.verticalLayout_2.addWidget(self.frame)
        self.frame_2 = QtWidgets.QFrame(self.groupBox)
        self.frame_2.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_2.setFrameShadow(QtWidgets.QFrame.Plain)
        self.frame_2.setObjectName("frame_2")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.frame_2)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.frame_4 = QtWidgets.QFrame(self.frame_2)
        self.frame_4.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_4.setFrameShadow(QtWidgets.QFrame.Plain)
        self.frame_4.setObjectName("frame_4")
        self.verticalLayout_4 = QtWidgets.QVBoxLayout(self.frame_4)
        self.verticalLayout_4.setObjectName("verticalLayout_4")
        self.label_6 = QtWidgets.QLabel(self.frame_4)
        self.label_6.setObjectName("label_6")
        self.verticalLayout_4.addWidget(self.label_6)
        self.label_5 = QtWidgets.QLabel(self.frame_4)
        self.label_5.setObjectName("label_5")
        self.verticalLayout_4.addWidget(self.label_5)
        self.label_4 = QtWidgets.QLabel(self.frame_4)
        self.label_4.setObjectName("label_4")
        self.verticalLayout_4.addWidget(self.label_4)
        self.horizontalLayout.addWidget(self.frame_4)
        self.frame_3 = QtWidgets.QFrame(self.frame_2)
        self.frame_3.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_3.setFrameShadow(QtWidgets.QFrame.Plain)
        self.frame_3.setObjectName("frame_3")
        self.verticalLayout_6 = QtWidgets.QVBoxLayout(self.frame_3)
        self.verticalLayout_6.setObjectName("verticalLayout_6")
        self.text_act1 = QtWidgets.QLineEdit(self.frame_3)
        self.text_act1.setObjectName("text_act1")
        self.verticalLayout_6.addWidget(self.text_act1)
        self.text_act2 = QtWidgets.QLineEdit(self.frame_3)
        self.text_act2.setObjectName("text_act2")
        self.verticalLayout_6.addWidget(self.text_act2)
        self.text_act3 = QtWidgets.QLineEdit(self.frame_3)
        self.text_act3.setObjectName("text_act3")
        self.verticalLayout_6.addWidget(self.text_act3)
        self.horizontalLayout.addWidget(self.frame_3)
        self.frame_5 = QtWidgets.QFrame(self.frame_2)
        self.frame_5.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_5.setFrameShadow(QtWidgets.QFrame.Plain)
        self.frame_5.setObjectName("frame_5")
        self.verticalLayout_5 = QtWidgets.QVBoxLayout(self.frame_5)
        self.verticalLayout_5.setObjectName("verticalLayout_5")
        self.label_3 = QtWidgets.QLabel(self.frame_5)
        self.label_3.setObjectName("label_3")
        self.verticalLayout_5.addWidget(self.label_3)
        self.label_7 = QtWidgets.QLabel(self.frame_5)
        self.label_7.setObjectName("label_7")
        self.verticalLayout_5.addWidget(self.label_7)
        self.label_8 = QtWidgets.QLabel(self.frame_5)
        self.label_8.setObjectName("label_8")
        self.verticalLayout_5.addWidget(self.label_8)
        self.horizontalLayout.addWidget(self.frame_5)
        self.frame_6 = QtWidgets.QFrame(self.frame_2)
        self.frame_6.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_6.setFrameShadow(QtWidgets.QFrame.Plain)
        self.frame_6.setObjectName("frame_6")
        self.verticalLayout_7 = QtWidgets.QVBoxLayout(self.frame_6)
        self.verticalLayout_7.setObjectName("verticalLayout_7")
        self.text_act4 = QtWidgets.QLineEdit(self.frame_6)
        self.text_act4.setObjectName("text_act4")
        self.verticalLayout_7.addWidget(self.text_act4)
        self.text_act5 = QtWidgets.QLineEdit(self.frame_6)
        self.text_act5.setObjectName("text_act5")
        self.verticalLayout_7.addWidget(self.text_act5)
        self.text_act6 = QtWidgets.QLineEdit(self.frame_6)
        self.text_act6.setObjectName("text_act6")
        self.verticalLayout_7.addWidget(self.text_act6)
        self.horizontalLayout.addWidget(self.frame_6)
        self.verticalLayout_2.addWidget(self.frame_2)
        self.frame_13 = QtWidgets.QFrame(self.groupBox)
        self.frame_13.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_13.setFrameShadow(QtWidgets.QFrame.Plain)
        self.frame_13.setObjectName("frame_13")
        self.gridLayout_2 = QtWidgets.QGridLayout(self.frame_13)
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.label_10 = QtWidgets.QLabel(self.frame_13)
        self.label_10.setObjectName("label_10")
        self.gridLayout_2.addWidget(self.label_10, 1, 0, 1, 1)
        self.label_11 = QtWidgets.QLabel(self.frame_13)
        self.label_11.setObjectName("label_11")
        self.gridLayout_2.addWidget(self.label_11, 3, 0, 1, 1)
        self.label_13 = QtWidgets.QLabel(self.frame_13)
        self.label_13.setObjectName("label_13")
        self.gridLayout_2.addWidget(self.label_13, 5, 0, 1, 1)
        self.txt_post_post = QtWidgets.QLineEdit(self.frame_13)
        self.txt_post_post.setObjectName("txt_post_post")
        self.gridLayout_2.addWidget(self.txt_post_post, 4, 2, 1, 1)
        self.label = QtWidgets.QLabel(self.frame_13)
        self.label.setObjectName("label")
        self.gridLayout_2.addWidget(self.label, 2, 0, 1, 1)
        self.label_9 = QtWidgets.QLabel(self.frame_13)
        self.label_9.setAlignment(QtCore.Qt.AlignCenter)
        self.label_9.setObjectName("label_9")
        self.gridLayout_2.addWidget(self.label_9, 0, 1, 1, 2)
        self.txt_res = QtWidgets.QLineEdit(self.frame_13)
        self.txt_res.setObjectName("txt_res")
        self.gridLayout_2.addWidget(self.txt_res, 5, 2, 1, 1)
        self.txt_post_neg = QtWidgets.QLineEdit(self.frame_13)
        self.txt_post_neg.setObjectName("txt_post_neg")
        self.gridLayout_2.addWidget(self.txt_post_neg, 3, 2, 1, 1)
        self.label_12 = QtWidgets.QLabel(self.frame_13)
        self.label_12.setObjectName("label_12")
        self.gridLayout_2.addWidget(self.label_12, 4, 0, 1, 1)
        self.txt_priori_pos = QtWidgets.QLineEdit(self.frame_13)
        self.txt_priori_pos.setObjectName("txt_priori_pos")
        self.gridLayout_2.addWidget(self.txt_priori_pos, 1, 2, 1, 1)
        self.txt_priori_neg = QtWidgets.QLineEdit(self.frame_13)
        self.txt_priori_neg.setObjectName("txt_priori_neg")
        self.gridLayout_2.addWidget(self.txt_priori_neg, 2, 2, 1, 1)
        self.verticalLayout_2.addWidget(self.frame_13)
        self.frame_7 = QtWidgets.QFrame(self.groupBox)
        self.frame_7.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.frame_7.setFrameShadow(QtWidgets.QFrame.Plain)
        self.frame_7.setObjectName("frame_7")
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout(self.frame_7)
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        spacerItem = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_2.addItem(spacerItem)
        self.btn_pred = QtWidgets.QPushButton(self.frame_7)
        self.btn_pred.setObjectName("btn_pred")
        self.horizontalLayout_2.addWidget(self.btn_pred)
        spacerItem1 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_2.addItem(spacerItem1)
        self.verticalLayout_2.addWidget(self.frame_7)
        self.verticalLayout.addWidget(self.groupBox)

        self.retranslateUi(Form)
        QtCore.QMetaObject.connectSlotsByName(Form)

    def retranslateUi(self, Form):
        _translate = QtCore.QCoreApplication.translate
        Form.setWindowTitle(_translate("Form", "Form"))
        self.groupBox.setTitle(_translate("Form", "Datos a predecir"))
        self.label_2.setText(_translate("Form", "Para cada actividad, introduce \"A\" si aprobo, \"R\" si reprobo o \"NP\" si no presento."))
        self.label_6.setText(_translate("Form", "Actividad 1"))
        self.label_5.setText(_translate("Form", "Actividad 2"))
        self.label_4.setText(_translate("Form", "Actividad 3"))
        self.label_3.setText(_translate("Form", "Actividad 4"))
        self.label_7.setText(_translate("Form", "Actividad 5 "))
        self.label_8.setText(_translate("Form", "Actividad 6"))
        self.label_10.setText(_translate("Form", "Probabilidad a priori (-):"))
        self.label_11.setText(_translate("Form", "Prob a posteriori (-):"))
        self.label_13.setText(_translate("Form", "Predicci√≥n: "))
        self.label.setText(_translate("Form", "Probabilidad a priori (+): "))
        self.label_9.setText(_translate("Form", "Resultados"))
        self.label_12.setText(_translate("Form", "Prob a posteriori (+):"))
        self.btn_pred.setText(_translate("Form", "predecir"))

